src/styles/about.styles.js

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+import { P } from "./styles";
+import { min, tablet } from "./breakpoints";
```

Comment:
`tablet` is an unused import, you can remove (you can also remove the line above it once you delete the `P` references)